favourites.html

```diff
@@ -54,6 +54,17 @@ <h2 class="ui header">
         </div>
       </div>
     </header>
+    <div class="ui grid favourite-card">
+      <div class="ui centered card">
+        <div class="image">
+          <img src="https://restcountries.eu/data/esp.svg" />
+        </div>
+        <div class="content ui grid col-4">
+          <a class="header">Spain</a>
```

Comment:
Can you center the header? I think it's center aligned or something :)

Reply:
"ui center aligned header"

Reply:
centered header for you =P